README.md

```diff
@@ -1,9 +1,10 @@
 <h1 align="center">
-  <img src="./images/logo-kaoto-dark.png">
+  <img src="./images/logo-kaoto-dark.png" alt="Kaoto">
 </h1>
 
 <p align="center">
   <a href="https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto"><img src="https://img.shields.io/visual-studio-marketplace/v/redhat.vscode-kaoto?style=for-the-badge" alt="Marketplace Version"/></a>
+  <a href="https://github.com/KaotoIO/kaoto-next/releases"><img alt="Kaoto UI version" src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2FKaotoIO%2Fvscode-kaoto%2Fmain%2Fpackage.json&query=%24.dependencies%5B'%40kaoto-next%2Fui'%5D&style=for-the-badge&label=Kaoto%20UI&color=orange"></a>
```

Comment:
Is it picking the version always be picked up from the main branch when rendering the file? if yes, it will give a wrong value on the Marketplace when it is updated on main branch but not yet released.

Reply:
uh oh.. good point 😱

... lets see if I can use current local version of package.json

Reply:
mmm I am thinking.. it is not possible to give weong version on marketplace even if you will merge into main.. on marketplace it should still contains version which was up to date when doing release.. if I remember correctly on marketplace readme there is not rendering or reloading connection

Reply:
so basically main and marketplace can have dofferent badge version for kaoto ui.. but it is not wrong, or am I missing something?

Reply:
ok, currently I cannot see way how to load dynamically context of local package.json 🙁 .. all options supporting only https api means we need to link some raw json from url..